tests/extra_code/implementations.cc

```diff
@@ -1 +1,6 @@
-bool {name}::lt(int i) const { return number() < i; }
```

Comment:
These should not be clang-formatted (and I think they should also be excluded from the pre-commit workflow)

Reply:
This should be already excluded from pre-commit right?
https://github.com/AIDASoft/podio/blob/0a5af60c203147de1e7f0081810e0b2fcfc6be5f/.pre-commit-config.yaml#L14
https://github.com/AIDASoft/podio/blob/0a5af60c203147de1e7f0081810e0b2fcfc6be5f/.pre-commit-config.yaml#L19

To exclude from `clang-format` run automatically by some IDE, adding a new file clang-format config should work.
`tests/extra_code/.clang-format` with:
```
DisableFormat: true
```
Would you like a separate PR or is it fine to do it here?

Reply:
I was not aware that is a possibility. Can you make a separate PR for that and also put one into the `python/templates` directory? Then I could probably remove the custom hook from my config :)

Reply:
Opened #605